alliance/apps/shared/models/team_project.py

```diff
@@ -12,7 +12,7 @@ class TeamProject(models.Model):
     claim_backlog = models.CharField(max_length=100, blank=True, null=True)
 
     def __unicode__(self):
-        return self.team, self.project
+        return '{}, {}'.format(self.team, self.project)
```

Comment:
Ah, this makes sense.

Once this gets addressed, we can go ahead and get this merged in.